src/sync/server.rs

```diff
@@ -251,14 +253,14 @@ fn check_method_handler_threads(ts: &ThreadS) {
 
 impl Default for Server {
     fn default() -> Self {
-        let (rfd, wfd) = pipe2(OFlag::O_CLOEXEC).unwrap();
         Server {
             listeners: Vec::with_capacity(1),
-            monitor_fd: (rfd, wfd),
+            monitor_fd: (0, 0),
```

Comment:
0 is not a good default value for file descriptions

Reply:
fixed